### DISCUSSION TITLE
Get corresponding account info with get_free_busy_info result when passing multiple accounts

### DISCUSSION THREAD

**Discussion by avin3sh** [ecederstrand/exchangelib#1204] [Q&A]
I am observing an issue where if I pass multiple accounts (<=100) to `get_free_busy_info`, sometimes the returned availability information does not contain data pertaining to one of the passed accounts -- this causes the order of all the items to shift by 1.

Is there a property in the returned availability information that I can use to map the result to the corresponding account instead of relying on the original order of accounts that was passed to `get_free_busy_info` ?

**Comment by ecederstrand** [comment]
The response objects in the XML response document should line up with the requested accounts in the request XML. Apparently, that doesn't happen.

Can you look into the XML documents (see https://ecederstrand.github.io/exchangelib/#troubleshooting) when requesting an account that doesn't return availability information? That should give is a hint as to why things don't line up.

**Reply by avin3sh** [reply]
It happens very rarely and difficult to reproduce outside of production environment but I will try.

**Reply by ecederstrand** [reply]
Thanks! I'm interested to see whether the response XML (see examples in https://learn.microsoft.com/en-us/exchange/client-developer/web-service-reference/getuseravailability-operation) contains as many `FreeBusyResponse` elements as there are are `MailboxData` elements in the request. If so, how does the `FreeBusyResponse` look in the failing case?
